src/style-library/components/pages/ImageScreen.scss

```
@@ -46,18 +52,19 @@
 	display: flex;
 	align-items: center;
 	justify-content: center;
-	width: 100%;
-	height: 100%;
+	// Fixed, and the reason the tile can grow: with the photo a known size and the fill sized
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)